lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -831,6 +831,50 @@ public static function get_style_variations_from_directory( $directory, $scope =
 		return $variations;
 	}
 
+	/**
+	 * Resolves relative paths in theme.json typography to theme absolute paths
+	 * and returns them in an array that can be embedded
+	 * as the value of `_link` object in REST API responses.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param 
+	 * @return array An array of resolved paths.
+	 */
+	private static function get_resolved_fonts_theme_uris( $theme_json_data ) {
+		$resolved_theme_uris = array();
+
+		if ( !empty( $theme_json_data['settings']['typography']['fontFamilies'] ) ) {
+
+			$font_families = ( $theme_json_data['settings']['typography']['fontFamilies']['theme'] ?? array() )
+				+ ( $theme_json_data['settings']['typography']['fontFamilies']['custom'] ?? array() )
```

Comment:
Was the union operator `+` intentional? I think `array_merge()` is more readable and will reindex the new array, avoid potential lost data.

See: https://stitcher.io/blog/array-merge-vs+

Unit test coverage would help test that.

Reply:
:+1: Using array_merge now